provision/acc_provision/acc_provision.py

```diff
@@ -151,6 +181,8 @@ def deep_merge(user, default):
 def config_default():
     # Default values for configuration
     default_config = {
+        "template_type": TEMPLATE_KUBE,
```

Comment:
should be "aci-containers.yaml" not "kube" right?

Reply:
Having a different template name is not enough. I think there are enough other differences in generating the output file that it deserves a separate function of its own. My plan is to make the output generation function a "flavor option"